src/plugin/ChainEthereumV1.ts

```diff
@@ -288,6 +296,9 @@ class Plugin implements Interfaces.Chain {
   /** Whether the chain supports resources */
   supportsResources = false
 
+  /** Whether chain supports ability to get a publicKey from a signature */
+  supportsTypedDataSignature = true
```

Comment:
Please rename this member to supportsSignMessageTypedData
Update the comment to be accurate - and please check all other comments in this PR